src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -135,6 +137,18 @@ public void updateFilteredPersonList(Predicate<Person> predicate) {
         filteredPersons.setPredicate(predicate);
     }
 
+    @Override
+    public void updateFriendList(Predicate<Person> predicate) {
+        requireNonNull(predicate);
+        friendList.setPredicate(combinedFriendPredicate(predicate, friendsPredicateFromPerson(user)));
+    }
+
+    @Override
+    public void updateOtherList(Predicate<Person> predicate) {
+        requireNonNull(predicate);
+        otherList.setPredicate(combinedOtherPredicate(predicate, othersPredicateFromPerson(user)));
+    }
+
     @Override
     public ObservableList<Person> getFriendList(Person person) {
```

Comment:
I think I mentioned this #56 as well but in this context it seems weird to have to pass a person to the API method, given that all it should do update the app with the current `User`'s friend list instead of any other `Person`'s. Allowing methods to pass any `Person` seems to be unsafe if some other method accidently calls it with anyone other than the current `User`.

More things to carry to v1.4 I guess.